### ISSUE TITLE
Downloading Darwin File

### ISSUE THREAD

**Issue by sean-hill** [officert/mongotron#64]
Chrome doesn't like it ...

<img width="361" alt="screen shot 2015-12-27 at 6 04 14 pm" src="https://cloud.githubusercontent.com/assets/1205866/12012828/4c935838-acc4-11e5-8f7f-d3a6047c24a1.png">

**Comment by officert** [comment]
Are you downloading from the Github release page?

https://github.com/officert/mongotron/releases

**Comment by nodkz** [comment]
yep, download via this page https://github.com/officert/mongotron/releases
by this link https://github.com/officert/mongotron/releases/download/v1.0.0-alpha.2/Mongotron-darwin-x64.zip

<img width="933" alt="screen shot 2015-12-28 at 10 00 01" src="https://cloud.githubusercontent.com/assets/1946920/12013951/2dd7653c-ad4c-11e5-8281-1d356c9e35e0.png">

**Comment by joebenson** [comment]
Same thing happens for me.

**Comment by joegyoung** [comment]
Chrome is blocking the download for https://github.com/officert/mongotron/releases/download/v1.0.0-alpha.2/Mongotron-darwin-x64.zip as nodkz displayed.

**Comment by AddoSolutions** [comment]
Same here.  Do we know if there is a way to check with Google to see why they are listing it as a virus?

**Comment by officert** [comment]
I haven't had time to look into this yet. Are you guys not able to download the release at all? If so I could try hosting it somewhere else and see if that helps.

**Comment by AddoSolutions** [comment]
Not sure if that is the problem, I doubt Google would have a problem with GitHub.

It can be manually extracted by renaming the file to a `.zip`file, but that makes me a bit nervous.

![image](https://cloud.githubusercontent.com/assets/3859209/12122516/8aafdc18-b3a9-11e5-9db7-8f069d0bce37.png)

**Comment by joebenson** [comment]
I agree with Addo. Can't download and use the release without the warning,
wish is a bit nerve wrecking. Haven't installed to try / contribute
specifically because of this, to be honest.

--
Joe Benson, CTO

Legnd
*Experience. Results.*

legnd.com
joe@legnd.com
(855) 695-3463

**Comment by officert** [comment]
To contribute you'll need to pull down the git repo which isn't affected by this. 
Though I do understand the hesitation, I just tried downloading and got the error message and it is a bit worrisome. I'm going to look into this some more and try and get it resolved. 
Would love the extra contributors, so please don't hesitate to jump in :+1:

**Comment by officert** [comment]
Just found this which could be related:

https://github.com/atom/atom/issues/3927

ran the darwin release through virus total for scanning for viruses:

https://www.virustotal.com/en/file/19f63f1f06574113ce25ed6b9f910ffa483d23a4fe33ff193d42bbf571cf119c/analysis/

1 thing detected:

<img width="1062" alt="screen shot 2016-01-05 at 2 23 45 pm" src="https://cloud.githubusercontent.com/assets/4620560/12125193/02688102-b3b8-11e5-9834-93bd1f7beef4.png">

**Comment by joebenson** [comment]
Tim, yep I'm familiar with the fork and pull requests, etc. I'll be happy
to contribute once things slow down a bit at work, and Google is sure the
file is safe ;)

Thanks for starting a great project.

--
Joe Benson, CTO

Legnd
*Experience. Results.*

legnd.com
joe@legnd.com
(855) 695-3463

**Comment by officert** [comment]
This seems to be fixed in the latest release:

https://github.com/officert/mongotron/releases

If anyone has any more problems with this please comment.
